frontend/ui/widget/sortwidget.lua

```diff
@@ -209,99 +177,89 @@ function SortWidget:init()
-
     -- group for footer
-    local footer_left_text = "◀"
-    local footer_right_text = "▶"
-    local footer_first_up_text = "◀◀"
-    local footer_last_down_text = "▶▶"
+    local chevron_left = "chevron.left"
+    local chevron_right = "chevron.right"
+    local chevron_first = "chevron.first"
+    local chevron_last = "chevron.last"
     if BD.mirroredUILayout() then
-        footer_left_text, footer_right_text = footer_right_text, footer_left_text
-        footer_first_up_text, footer_last_down_text = footer_last_down_text, footer_first_up_text
+        chevron_left, chevron_right = chevron_right, chevron_left
+        chevron_first, chevron_last = chevron_last, chevron_first
     end
     self.footer_left = Button:new{
-        text = footer_left_text,
-        width = self.width_widget * 13 / 100,
+        icon = chevron_left,
+        width = self.footer_button_width,
         callback = function() self:prevPage() end,
-        text_font_size = 28,
         bordersize = 0,
-        padding = 0,
         radius = 0,
         show_parent = self,
     }
     self.footer_right = Button:new{
-        text = footer_right_text,
-        width = self.width_widget * 13 / 100,
+        icon = chevron_right,
+        width = self.footer_button_width,
         callback = function() self:nextPage() end,
-        text_font_size = 28,
         bordersize = 0,
-        padding = 0,
         radius = 0,
         show_parent = self,
     }
     self.footer_first_up = Button:new{
-        text = footer_first_up_text,
-        width = self.width_widget * 13 / 100,
+        icon=chevron_first,
+        width = self.footer_button_width,
         callback = function()
             if self.marked > 0 then
                 self:moveItem(-1)
             else
                 self:goToPage(1)
             end
         end,
-        text_font_size = 28,
         bordersize = 0,
-        padding = 0,
         radius = 0,
         show_parent = self,
     }
     self.footer_last_down = Button:new{
-        text = footer_last_down_text,
-        width = self.width_widget * 13 / 100,
+        icon=chevron_last,
+        width = self.footer_button_width,
         callback = function()
             if self.marked > 0 then
                 self:moveItem(1)
             else
                 self:goToPage(self.pages)
             end
         end,
-        text_font_size = 28,
         bordersize = 0,
-        padding = 0,
         radius = 0,
         show_parent = self,
     }
     self.footer_cancel = Button:new{
-        text = "✘",
-        width = self.width_widget * 13 / 100,
+        text = "Cancel",
+        width = self.footer_button_width,
         callback = function() self:onClose() end,
         bordersize = 0,
-        text_font_size = 28,
         padding = 0,
         radius = 0,
         show_parent = self,
     }
-
     self.footer_ok = Button:new{
-        text= "✓",
-        width = self.width_widget * 13 / 100,
+        text= "Save",
```

Comment:
Shouldn't we use `_("Save")` here for translations?

Reply:
Done, thanks